auction-app-backend/src/main/java/com/example/auctionapp/entity/ProductImageEntity.java

```diff
@@ -25,7 +25,7 @@ public class ProductImageEntity {
     @Column(name = "image_url")
     private String imageUrl;
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Why you have added CascadeType.ALL?

Reply:
I meant to add that in my product entity but i accidentally added here at first, i forgot to remove it later on